pyk/src/pyk/tests/test_kcfg.py

```diff
@@ -32,10 +38,9 @@ def node_dicts(n: int) -> List[Dict[str, Any]]:
 
 
 def edge_dicts(*edges: Tuple[int, int]) -> List[Dict[str, Any]]:
-    return [
-        {'source': nid(i), 'target': nid(j), 'condition': TRUE.to_dict(), 'depth': 1}
-        for i, j in edges
-    ]
+    def make_edge_dict(i, j, depth=1):
+        return {'source': nid(i), 'target': nid(j), 'condition': TRUE.to_dict(), 'depth': depth}
+    return [make_edge_dict(*edge) for edge in edges]
```

Comment:
```suggestion

    def _make_edge_dict(i, j, depth=1):
        return {'source': nid(i), 'target': nid(j), 'condition': TRUE.to_dict(), 'depth': depth}
        
    return [_make_edge_dict(*edge) for edge in edges]
```

I'm pretty sure our codestyle enforces the space around internally defined functions (maybe this repos isn't as strict as ERC20s?). But we have been naming internally defined functions with `_...` prefixes as a convention.